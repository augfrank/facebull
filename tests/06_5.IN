M0   C0   C2  2216
M1   C0   C3  2345
M2   C0   C5  2308
M3   C1   C2  2026
M4   C1   C3  2570
M5   C1   C4  2953
M6   C2   C0  2400
M7   C2   C1  2268
M8   C2   C3  2491
M9   C3   C0  2231
M10   C3   C2  2209
M11   C3   C4  2759
M12   C3   C5  2000
M13   C4   C2  2892
M14   C4   C5  2640
M15   C5   C0  2424
M16   C5   C1  2224
M17   C5   C2  2115
M18   C5   C4  2836
