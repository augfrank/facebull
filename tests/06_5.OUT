14394
1 3 6 11 14 16
